delivery_postlogistics/models/stock_picking.py

```diff
@@ -75,26 +75,34 @@ def attach_shipping_label(self, label):
             .create(data)
         )
 
+    # When base_delivery_carrier_label is installed, its _set_a_default_package
+    # is called instead of this module's one. So the logic should be moved to
+    # _put_in_pack.
+    # TODO 15.0: refactor to depends on base_delivery_carrier_label
```

Comment:
BTW: We definitely do not want to depend anymore on `base_delivery_carrier_label` for this module as it used to be the case before.